### PR TITLE
SAK-42179 Improve graphical display of test statistics

### PR DESCRIPTION
**Pull request by dnhutchins**

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 18
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

File: library/pom.xml [hunk 1]
```diff
@@ -159,6 +159,12 @@
 			<version>1.2.2</version>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.webjars</groupId>
+			<artifactId>jqplot</artifactId>
+			<version>1.0.8r1250</version>
```
Comment (ottenhoff) [inline library/pom.xml]: May 2013 dependency ... maybe doesn't matter ..... does 1.0.9.d96a669-1 version work?

**Comment by ottenhoff** [comment]
I'm seeing an empty Final Score Distribution ..... no console errors

The question score section looks great. Thanks for this contribution.

Screen Shot 2020-02-12 at 3 02 11 PM

File: library/pom.xml (hunk 1 above)
Comment (dnhutchins) [reply library/pom.xml]: I will try and see. I suspect it would work fine.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

**Comment by dnhutchins** [comment]
I tweaked a couple more things, and upgraded jqplot, which seems to work fine.

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 12
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 49
           

Complexity increasing per file
==============================
- samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/HistogramScoresBean.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/sakaiproject/sakai/pullRequest?prid=4993556&bid=16482207)

**Pull request merged**